libs/design-system/cards/content-button/src/lib/content-button.schema.ts

```diff
@@ -0,0 +1,16 @@
+import { ContentTemplateSchema } from '@hra-ui/cdk/content-template';
+import { z } from 'zod';
```

Comment:
Update import to `import * as z from 'zod'. More tree shakeable since we upgraded to version 4 of zod